### PR TITLE
feat: add exception for partner IAM (#767)

### PR DESCRIPTION
**Pull request by tkrop**
Adds an exception text for scopes in partner IAM.

Contributes to #767.

### CODE REVIEW

File: chapters/security.adoc [hunk 1]
```diff
@@ -152,6 +152,11 @@ permission names in APIs must conform to the following naming pattern:
 
 This pattern is compatible with the previous definition.
 
+**Note:** This naming convention only applies to scopes for service-to-service
+communication using the Platform IAM tokens. For IAM systems with other naming
+rules (e.g. Zalando Partner IAM), the naming convention is applied consistent
+with the naming conventions of this IAM system.
```
Comment (SmilyOrg) [inline chapters/security.adoc:158]: Rephrased a bit, feel free to reject if it doesn't seem clearer.
```suggestion
**Note:** This naming convention only applies to scopes for service-to-service
communication using the Platform IAM tokens. For IAM systems with other naming
rules (e.g. Zalando Partner IAM), the naming should be consistent with the
existing conventions of those systems.
```

**Comment by SmilyOrg** [comment]
:+1:

**Comment by tkrop** [comment]
👍

**Comment by ktsypkina** [comment]
👍

**Pull request merged**